commands/viewer_commands.py

```diff
@@ -141,26 +141,37 @@ async def bet(
             Choice(name="Down", value="Down"),
             Choice(name="R", value="R"),
             Choice(name="L", value="L"),
+        ],
+        amount=[
```

Comment:
Use `Optional[Range[int, 1, 9]]` on a number instead of this, this is overcomplicated.
Something like this:
> async def pokemon_move(self, interaction: Interaction, move: str, amount: Optional[Range[int, 1, 9]] = 1):

Range needs to be imported from app_commands, Optional from typing.
Also please add `@app_commands.describe` for the arguments. The description for amount should include the 1-9 limit.